arch/arm/boot/dts/overlays/README

```diff
@@ -3687,10 +3732,8 @@ Params: <None>
 
 
 Name:   vc4-kms-kippah-7inch
-Info:   Enable the Adafruit DPI Kippah with the 7" Ontat panel attached.
-        Requires vc4-kms-v3d to be loaded.
-Load:   dtoverlay=vc4-kms-kippah-7inch
-Params: <None>
+Info:   This overlay is now deprecated - see i2c-sensor
```

Comment:
See vc4-kms-dpi-panel, surely.

Reply:
:-( Guess where the copy/paste came from! Will fix it up when I'm in the office.